superset-frontend/src/components/ListView/utils.ts

```diff
@@ -63,11 +68,11 @@ function updateInList(list: any[], index: number, update: any): any[] {
   ];
 }
 
-function mergeCreateFilterValues(list: Filter[], updateList: FilterValue[]) {
+function mergeCreateFilterValues(list: Filter[], updateObj: any) {
```

Comment:
can we type the query filters object? something like 
```
type QueryFilterState = {
  [id: string]: [value: FilterValue["value"], operator: FilterValue["value"]]
}
```

Reply:
I'm also going to be modifying this file in https://github.com/apache/incubator-superset/pull/11702 so I'll make this change when I rebase after this is merged.